source/faucet/config.d

```diff
@@ -35,6 +35,19 @@ public struct Config
         this.tx_generator = tx_generator;
         this.web = web;
     }
+
+    /// We do not want to log the key seeds
+    string toString ()
+    {
+        return format!"Web(address:%s, post:%s),"
+            (this.web.address, this.web.port) ~
+            format!"TxGenerator(send_interval:%s, split_count:%s, merge_threshold:%s, addresses:%s, stats_port:%s)"
+            (this.tx_generator.send_interval,
+            this.tx_generator.split_count,
+            this.tx_generator.merge_threshold,
+            this.tx_generator.addresses,
+            this.tx_generator.stats_port);
```

Comment:
I think that next time we add a config option, we'll forget this.
I'll suggest either of:
1) Store `KeyPair` (or `SecretKey`) in the file so you use the overriden `toString`;
2) Do not log the config;
3) If we go with this, add a `tupleof` check:
```
static assert(Config.tupleof.length == X, "`Config.toString()` needs an update");
```

Reply:
A went with something similar to option 1 but with a new struct named `Seeds` which logs the number of keys rather than the values.